### PR TITLE
WIP KG

### PR DESCRIPTION
**Pull request by TheObserverofOne**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,153 +17,263 @@
 </head>
 
 <body>
-  <div class="container-fluid">
+  <div class="container-fluid sticky-top">
```
Comment (lefrenk) [inline index.html]: 👍

File: index.html [hunk 2]
```diff
@@ -17,153 +17,263 @@
...
         </div>
-        <div class="col-9">
-          <nav class="nav">
-            <a class="nav-link active" href="#">Home</a>
-            <a class="nav-link" href="#">About Us</a>
-            <a class="nav-link" href="#">Blog</a>
-            <a class="nav-link" href="#">Contact</a>
-          </nav>
+        <div class="col-9 d-flex justify-content-end align-items-center">
+            <!-- JavaScript Component: Navbar and Dropdown -->
```
Comment (lefrenk) [inline index.html]: Your navbar dropdown menu looks really good!

File: index.html [hunk 3]
```diff
@@ -17,153 +17,263 @@
...
             <h1 class="logo-heading">Fun Bus</h1>
         </div>
-        <div class="col-9">
-          <nav class="nav">
-            <a class="nav-link active" href="#">Home</a>
-            <a class="nav-link" href="#">About Us</a>
-            <a class="nav-link" href="#">Blog</a>
-            <a class="nav-link" href="#">Contact</a>
-          </nav>
+        <div class="col-9 d-flex justify-content-end align-items-center">
```
Comment (lefrenk) [inline index.html]: Excellent use of utilities

File: index.html [hunk 4]
```diff
@@ -17,153 +17,263 @@
...
-        <p>Excursion clean modern design wanderlust, traveler wanderlust expedition WordPress whimsical WordPress website whimsical wanderlust. Blogger wanderlust modern modern design modern colorful simple clean. Wanderlust Travel adventure, traveler excursion wanderlust website WordPress adventure darn website darn.</p>
+      <div class="row">
+          <div class="col-sm-4 mb-4">
+              <h4>Fun In The Sun</h4>
+              <p>Expedition excursion design darn excursion fun, clean simple organized WordPress Travel colorful webdesign. Traveler blogger website design expedition clean excursion traveling.</p>
+              <button type="button" class="btn btn-info" data-toggle="modal" data-target="#exampleModal">
+                  Get More Info
+              </button>
+
+              <!-- JavaScript Component: Modal -->
```
Comment (lefrenk) [inline index.html]: Awesome!

**Comment by lefrenk** [comment]
Very good work, Kev!👌 You clearly have a solid grasp of Bootstrap

— Site looks great ✔️
— Code is consistent with HTML/CSS idioms and style ✔️
— Demonstrated understanding of the utilities, components, forms, grid sytem ✔️

Please close this PR after reviewing feedback. Thank you

**Comment by TheObserverofOne** [comment]
Great!~

**Pull request closed without merging**